src/main/java/com/example/notes/entity/NoteEntity.java

```diff
@@ -0,0 +1,20 @@
+package com.example.notes.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "notes")
+@Data
+public class NoteEntity {
```

Comment:
We also may need @EqualsAndHashcode + @ToString on top of every entity

Reply:
@EqualsAndHashCode + @tostring are included in @Data, so there are already here